app/instance-initializers/ember-i18n.js

```diff
@@ -14,6 +14,7 @@ export default {
     const key = 'service:i18n';
     const i18n = instance.lookup ? instance.lookup(key) : instance.container.lookup(key);
     i18n.set('locale', defaultLocale);
+    i18n.set('allowLocaleOverride', (ENV.i18n || {}).allowLocaleOverride);
```

Comment:
I think it's probably a good idea that all the code which initializes the service with ENV values should be in the same place – it would be strange to have one property over here, and one property over there. Or would you also suggest moving the `defaultLocale` stuff? (that seems like a separate change)